view/adminhtml/templates/cms/wordpressdata.phtml

```diff
@@ -0,0 +1,18 @@
+<?php
+/** @var $viewModel \Mooore\WordpressIntegrationCms\ViewModel\WPCIData */
+
+$viewModel = $block->getViewModel();
+
+$wordPressUrl = $viewModel->getWordPressUrl();
+?>
+<?php if ($wordPressUrl):?>
+<script>
+    require([
+        'jquery'
+    ], function($){
+        $(document).ready(function() {
+            window.wordpressurl = '<?php echo $wordPressUrl; ?>';
```

Comment:
What will happen when multiple CMS-envs are set to the installation? Will it only use one to calculate all editor URLs with?